### PR TITLE
Add "data_" to metric file names to fix uBlock Origin blocking JSON calls

### LINKED ISSUE #550: Ad blocker blocks metric names that start with `metrics`
Metric names that start with `metrics`, such as:

[metrics.activity_state_provider_error](https://dictionary.protosaur.dev/apps/fenix/metrics/metrics_activity_state_provider_error)

[metrics.adjust_network](https://dictionary.protosaur.dev/apps/fenix/metrics/metrics_adjust_network)

[metrics.distribution_id](https://dictionary.protosaur.dev/apps/fenix/metrics/metrics_distribution_id)

are currently blocked by uBlock Origin. 

CleanShot 2021-04-26 at 08 51 27@2x

CleanShot 2021-04-26 at 08 51 49@2x

Looks like it doesn't like the google analytics tracker that we use, however I'm not sure why it only targets metric name that starts with `metrics`. Other metric pages seem to work fine.

### PR DESCRIPTION
**Pull request by Iinh**
Fixes #550.
### Pull Request checklist

- [x] The pull request has a descriptive title (and a reference to an issue it
      fixes, if applicable)
- [x] All tests and linter checks are passing
- [x] The pull request is free of merge conflicts

### CODE REVIEW

**Comment by Iinh** [comment]
Pages of `metrics_` metrics should load fine now even with uBlock origin on.

e.g. [metrics.activity_state_provider_error](https://deploy-preview-553--glean-dictionary-dev.netlify.app/apps/fenix/metrics/metrics_activity_state_provider_error), [metrics.adjust_network](https://deploy-preview-553--glean-dictionary-dev.netlify.app/apps/fenix/metrics/metrics_adjust_network), [metrics.distribution_id](https://deploy-preview-553--glean-dictionary-dev.netlify.app/apps/fenix/metrics/metrics_distribution_id)

File: scripts/build-glean-metadata.py [hunk 1]
```diff
@@ -18,6 +18,21 @@ def _serialize_sets(obj):
...
+    # replace . with _ so sirv doesn't think that
+    # a metric is a file
+    metric_name = name.replace(".", "_")
+
+    # if a metric name starts with "metrics", uBlock Origin
+    # will block the network call to get the JSON resource
+    # See: https://github.com/mozilla/glean-dictionary/issues/550
+    # To get around this, we add "data" to said metric names
+    if metric_name.split("_")[0] == "metrics":
+        return f"data_{metric_name}"
```
Comment (wlach) [inline scripts/build-glean-metadata.py:31]: It's probably easier to just prefix everything with `data_`. That way we don't need conditional logic here.

File: src/__snapshots__/storyshots.test.js.snap [hunk 2]
```diff
@@ -5,13 +5,13 @@ exports[`Storyshots App Alert Text 1`] = `
   class="storybook-snapshot-container"
```
Comment (wlach) [inline src/__snapshots__/storyshots.test.js.snap:5]: My guess is that you might have some unrelated changes locally which aren't part of this commit. I would just revert these changes-- nothing in this PR should change the storyshots.

**Review by wlach — CHANGES_REQUESTED** [review]
Looks reasonable for the most part! I think the most obvious solution to the errors is just to revert the storyshot changes (I'm curious why regenerating the storyshots would change them if you don't have anything else in your local checkout...)

**Review by wlach — APPROVED** [review]

**Comment by wlach** [comment]
Thanks @Iinh!
